packages/cli/src/internal/options.ts

```diff
@@ -1144,6 +1144,9 @@ const toParseableInstruction = (self: Instruction): Array<ParseableInstruction>
   }
 }
 
+/** @internal */
+const keyValueSplitter = new RegExp('=(.*)')
```

Comment:
```suggestion
const keyValueSplitter = /=(.*)/
```

@dpnova - sorry for the delay here - can we update this to just a plain regular expression? No need for the constructor. Then I'll go ahead and get this merged.

Reply:
thanks @IMax153 - done. I recalled seeing something in tselint stylistic about preferring the constructor... never really understood why though haha.